pkgs/development/python-modules/vinput/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+  setuptools,
+  wheel,
+  testers,
+  vinput,
+}:
+
+buildPythonPackage rec {
+  pname = "vinput";
+  version = "0.6.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "xslendix";
+    repo = "libvinput.py";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-86oNsbdw59zvxQvn05Q493mAoZdIBWUu/GtQvIJiEEY=";
+  };
+
+  postPatch = ''
+    substituteInPlace "vinput/_binding.py" \
+      --replace-fail  "dir_path = os.path.dirname(os.path.realpath(__file__))" "dir_path = "$out/""
+  '';
+
+  build-system = [
+    setuptools
+    wheel
+  ];
+
```

Comment:
Are there any tests? If so, please add pytestCheckHook to nativeCheckInputs

Reply:
Build fails with 0 tests executed, there a test.py file but seems it is not referenced in pyproject.toml